packit/distgit.py

```diff
@@ -265,13 +264,13 @@ def upload_to_lookaside_cache(self, archive_path: str) -> None:
         )
         f.init_ticket(self.config.keytab_path)
         try:
-            f.new_sources(sources=archive_path)
+            f.new_sources(sources=str(archive_path))
         except Exception as ex:
             logger.error(f"'fedpkg new-sources' failed for the following reason: {ex}")
             raise PackitException(ex)
 
-    def is_archive_in_lookaside_cache(self, archive_path: str) -> bool:
-        archive_name = os.path.basename(archive_path)
+    def is_archive_in_lookaside_cache(self, archive_path: Path) -> bool:
+        archive_name = archive_path.name
```

Comment:
This is just a detail, but you can rename `archive_path` to just `archive` since we now know it's a `Path`.
And you can remove the `archive_name` variable and use `archive.name` directly.